### PR TITLE
fix: themes in System: Gateways: Status page

### PR DESCRIPTION
**Pull request by fabianfrz**
ref: opnsense/plugins#508

don't use hardcoded colors in status_gateway.php

requested by @marjohn56

### CODE REVIEW

File: src/www/status_gateways.php [hunk 1]
```diff
@@ -85,29 +85,29 @@
...
+                            $gateway_label_class = "success";  // green
                         }
                     } else if (isset($gateway['monitor_disable'])) {
                         $online = gettext("Online");
-                        $bgcolor = "#90EE90";  // lightgreen
+                        $gateway_label_class = "success";  // green
                     } else {
                         $online = gettext("Pending");
-                        $bgcolor = "#D3D3D3";  // lightgray
+                        $gateway_label_class = "info";  // blue (no other color available in bootstrap 3 - its ok but would prefer gray)
```
Comment (fichtner) [inline src/www/status_gateways.php]: Default is not grey? https://getbootstrap.com/docs/3.3/components/#labels

Comment (fabianfrz) [reply src/www/status_gateways.php]: yes that worked - thanks

**Comment by marjohn56** [comment]
That certainly works for online and offline, biut I am unsure as to the effect on the rebellion theme as it's grey anyway...

**Comment by fichtner** [comment]
@marjohn56 this is where you want to redefine default, success, danger, etc to fit the theme perfectly and all the pages behave accordingly :)

File: src/www/status_gateways.php (hunk 1 above)
Comment (fichtner) [reply src/www/status_gateways.php]: can we drop all the // colour annotations? they are not necessarily true if modified in the theme

**Comment by marjohn56** [comment]
I thought that.. I thought Green for Danger, Red for OK and blue for default. :)

**Comment by fichtner** [comment]
@marjohn56 haha, your theme, your rules :)

**Comment by marjohn56** [comment]
I'm going to tone down the green a little, the others should not be too far out, although info looks like it could do with a little reduction too.

$brand-success:         #9BD275 !default;
$brand-info:               #B0CDDB !default;
$brand-warning:        #f0ad4e !default;
$brand-danger:          #F05050 !default;

**Comment by fichtner** [comment]
Merged, thanks. I'll remove the superfluous comments inline.

**Pull request merged**